requirements-dev.txt

```diff
@@ -4,6 +4,7 @@ docker>=3.7.0,<3.8.0
 freezegun>=1.1.0
 honcho>=1.1.0
 openapi-core>=0.14.2
+jsondiff>=2.0.0
```

Comment:
I'm not sure this provides much value over using `difflib` and producing a unified diff of the pretty printed json

Reply:
I've changed to using `difflib`, per your suggestion. I imported `JSONEncoder` and `better_default_encoded` from `setnry.util.json`, since the default override of the `json_dumps` method in that file does not seem to support pretty printing.